src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -158,6 +158,7 @@ def _get_base_properties(self):
             'Context.Default.VS.Core.Distro.Id': _get_distro_id(),  # eg. 'centos'
             'Context.Default.VS.Core.Distro.Version': _get_distro_version(),  # eg. '8.4.2105'
             'Context.Dafault.VS.Core.Istty': str(sys.stdin.isatty()),
+            'Context.Default.VS.Core.User.DeviceId': _get_device_id(),
```

Comment:
If you wish to align with VS, our property is `context.default.vs.core.devdeviceid`

Reply:
sounds good